.github/workflows/ci.yml

```
@@ -176,6 +176,12 @@ jobs:
             transport_arguments: -t serial-usb
             binary: 'firmware/build/zephyr/zephyr.bin'
 
+          - rtos: zephyr
+            platform: nucleo_h743zi
+            configuration: int32_publisher
+            transport_arguments: -t serial -d 3
+            binary: 'firmware/build/zephyr/zephyr.exe'
```

Typo here, I guess. This should fix the CI

```suggestion
            binary: 'firmware/build/zephyr/zephyr.bin'
```